pkg/controller/node/cidr_allocator.go

```diff
@@ -17,144 +17,169 @@ limitations under the License.
 package node
 
 import (
-	"encoding/binary"
 	"errors"
 	"fmt"
-	"math/big"
 	"net"
-	"sync"
+
+	"k8s.io/kubernetes/pkg/api"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/client/record"
+	"k8s.io/kubernetes/pkg/util/wait"
+
+	"github.com/golang/glog"
+)
+
+const (
+	// controls how many NodeSpec updates NC can process in any moment.
+	cidrUpdateWorkers   = 10
+	cidrUpdateQueueSize = 5000
```

Comment:
Is there any rationale behind this number? If so, can you please add a comment where does this number come from?